src/renderer/viz/expressions/base.js

```diff
@@ -65,19 +65,30 @@ export default class Base {
      * // `color` will have the same color as the features with an amount of 123
      *
      */
-    eval (feature) {
-        throw new CartoRuntimeError('Unimplemented');
+    eval () {
+        return this.value;
     }
 
     /**
-     * Get the expression value
+     * Get the expression value. It gets the data from the `getLegendData` method.
      *
      * @api
      * @memberof Expression
      * @name value
-     */
+    */
     get value () {
-        return this.eval();
+        throw new CartoRuntimeError('Must evaluate a feature using .eval(feature)');
+    }
+
+    /**
+     * Get the expression values
+     *
+     * @api
+     * @memberof Expression
+     * @name values
+     */
+    get values () {
+        return this.getLegendData().data;
```

Comment:
Does this always guarantee a result? Should we consider a runtime error + a proper message?